cron/jobstats.go

```diff
@@ -0,0 +1,117 @@
+package cron
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+type Job struct {
+	Category	string
+	Status		string
+	Count		int
+}
+
+//jobs/submitted
+func GetSubmittedJobCounts(db *sql.DB, startDay string, endDay string)([]Job, error) {
```

Comment:
In the functions in this file, you're using `startDay` and `endDay`, but elsewhere you're using `startDate` and `endDate`, with the same format. Better if these are consistent, otherwise it can be unclear if they're supposed to take differently-formatted information.